src/app/components/algorithms/impl-selection-criteria/impl-selection-criteria.component.html

```diff
@@ -1,97 +1,143 @@
 <h2>Required information for the NISQ Analyzer</h2>
-<div class="d-flex justify-content-between header-container">
-  <h4>Input Parameters</h4>
-  <div class="d-flex">
-    <button
-      mat-button
-      matSuffix
-      mat-icon-button
-      aria-label="Add Input Parameter"
-      (click)="addOne()"
-    >
-      <mat-icon>add</mat-icon>
-    </button>
-    <button
-      *ngIf="!selection.isEmpty()"
-      mat-button
-      matSuffix
-      mat-icon-button
-      aria-label="Remove selected parameters"
-      (click)="deleteMany()"
-    >
-      <mat-icon>remove</mat-icon>
-    </button>
-  </div>
+<div
+  *ngIf="!nisqImpl"
+  class="d-flex align-content-center justify-content-center"
+>
+  <mat-spinner></mat-spinner>
 </div>
-<div *ngFor="let param of params; let i = index">
+<div *ngIf="nisqImpl">
+  <div
+    class="d-flex justify-content-between align-self-center header-container"
+  >
+    <app-text-input
+      class="w-25 p-2"
```

Comment:
Isn't this also 25% on small screens? Maybe change this too what is done for other inputs. 
There are a few places where w-25 is used for inputs in this component.

Reply:
We had the same thing with w-50 on the selection criteria of the algorithms. Maybe check how it was adjusted in the closed responsiveness PR (https://github.com/UST-QuAntiL/qc-atlas-ui/pull/67/files). Change it with cols so on large screens it is 50% or 25% and on small 100%